package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yarn",
-  "version": "0.16.0",
+  "version": "0.16.2",
```

Comment:
We should get into the habit of bumping this after every release... The dev build number should always be greater than the release build number. We can later swap out the version number if we do a minor release (eg. 0.16.2 -> 0.17.0)